BUILD.md

```diff
@@ -24,3 +24,17 @@ tito tag
 ```
 tito build --rpm
 ```
+
+## Build openshift-ansible container image
+
+To build a container image of `openshift-ansible`:
+
+* Using standalone **Docker**:
+
+        cd openshift-ansible
+        docker build -t openshift/openshift-ansible .
+
+* Using an **OpenShift** build:
+
+        oc new-build docker.io/aweiteka/playbook2image~https://github.com/openshift/openshift-ansible
+        oc describe imagestream openshift-ansible
```

Comment:
I added the `oc describe is` to show how to find the built image... I thought that the `oc new-build` alone was missing some information/feedback... probably we could add some more info around it? It's only that an explanation of how OpenShift builds and imagestreams work is out of scope here.

I'll just add a link to the OpenShift docs around that if that's ok.